### ISSUE TITLE
Border/Outline bars

### ISSUE THREAD

**Issue by niallmc** [ricardodorosario/react-horizontal-stacked-bar-chart#9]
It would be good if there was a way outline the bars and each stack within the bar.

In my use case I want each of the stacks within a bar to be the same color, and an outline border of black around each stack to differentiate each one rather than the color.

It may be good to have something like the following properties:

outline: boolean
outlineColor: hex color
outlineWidth: number in pixels

**Comment by ricardodorosario** [comment]
Hi @niallmc ,
Thank you for the suggestion. It was noted and I will make ASAP.

But for that moment, you could use some CSS properties to change the colors of the component.

For instance:
```
svg#hsbar > g > rect {
  fill: white;
  stroke: black;
  stroke-width: 3px;
}
```

**Comment by niallmc** [comment]
@ricardodorosario Thanks for the CSS I can use for the moment.

**Comment by ricardodorosario** [comment]
Solved in #10
